docs/semgrep-secrets/rules.md

```diff
@@ -188,6 +216,15 @@ response is expected for valid and invalid states. Although some rules do not us
 See [<i class="fa-regular fa-file-lines"></i> Validators](/semgrep-secrets/validators/) for more information.
 :::
 
+## Metavariable binding
+
+Semgrep secrets uses metavariables. Metavariables allow Semgrep Secrets to reuse matched information from your code in its validators. An example of a metavariable is as follows:
+
+<iframe title="Message displays metavariable content" src="https://semgrep.dev/embed/editor?snippet=JDzRR" width="100%" height="432px" frameBorder="0"></iframe>
+<br />
+
+When you click **Run**, the content from the metavariable `$HELLO` displays as `This content is now reusable in validators`. If this were a secret, Semgrep Secrets could use this to call the appropriate service to determine if the secret is active.
```

Comment:
```suggestion
When you click **Run**, the content from the metavariable `$HELLO` displays as `This content is now reusable in validators`. If this is a Secret rule, Semgrep Secrets could use this to call the appropriate service to determine if the secret is active.
```
Not sure if I understood the intent of the sentence, just tried to make it clear.